app/assets/v2/images/avatar3d/avatar_bufficorn.svg

```diff
@@ -0,0 +1,8062 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 23.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"

```

Comment:
be careful with this svg and our svg optimizer because will remove all the css, I recommend to try this tool and check what you get if is fine use it instead. https://jakearchibald.github.io/svgomg/